### ISSUE TITLE
Sauce didn't detect riders around me

### ISSUE THREAD

**Issue by gtdean11** [SauceLLC/sauce4zwift#93]
**Describe the bug**
Rode one of the new routes (canopies and coastlines) in non-race and for the first part of the ride it couldn't detect riders around me or place me on the map.  Interestingly, once I got on the new section things worked fine.

**Desktop (please complete the following information):**
 - Windows

**Additional context**
Haven't tried to recreate and unfortunately didn't record

**Comment by mayfield** [comment]
This is sort of as-intended.  The route data in the game files is not perfectly inclusive of the start and end of where a player might be teleported to when starting a route.

**Comment by gtdean11** [comment]
It wasnt just the first few meters. It was for a solid 5-10km

**Comment by mayfield** [comment]
@gtdean11 was this on the alpha build?

**Comment by gtdean11** [comment]
yes was on alpha.  I'll record more often to see if I can recreate.
